.circleci/config.yml

```diff
@@ -66,12 +66,14 @@ jobs:
     steps:
       - checkout
       - setup_remote_docker:
-          version: 17.07.0-ce
+          version: 18.06.0-ce
       - attach_workspace:
           at: .
-      - run: ln -s .build/linux-amd64/thanos thanos
-      - run: make docker
-      - run: docker run thanos --help
+      # register qemu to support multiple architectures build
+      - run: docker run --privileged linuxkit/binfmt:v0.7
+      - run: make docker-build
```

Comment:
I suspect it won't work because Circle CI doesn't cross-build binaries when merging on `master`.